FranciscoPerez/react-challenge-project-jan-2023
# Fp 10 add user to nav

## Changes
1. Add user email to nav.

## Purpose
There is no indication in the app regarding login status.

[Issue #10](https://github.com/Shift3/react-challenge-project-jan-2023/issues/10)

## Approach
Use the `useSelector` hook to select and extract the email from the auth redux state. The email can then be added to the nav to indicate which user is logged in.

## Testing Steps
1. Navigate to the `/login` page.
2. Enter email and password.
3. Click on the Login button.
4. Verify that the email is displayed at the top of the nav.

## Learning
It was a straightforward use of the `useSelector` hook so no research was required in this case.

Closes #10
